website/static/js/search.js

```diff
@@ -169,7 +169,7 @@ var ViewModel = function(params) {
         }
     };
 
-    self.addTag = function(name) {
+    self.addTag = function(name, action) {
```

Comment:
Having an `addTag` where `action` can be `remove` is untuitive. Why not a `removeTag` method?